### ISSUE TITLE
IGV batch mode hangs on goto use

### ISSUE THREAD

**Issue by javiercguard** [igvteam/igv#900]
Hi, IGV is a fantastic tool, but I've found a problem when using the batch mode: IGV can't complete `goto` commands when the argument is other than `all`.
I've also noticed something else: the previous problem does not happen when there isn't a `prefs.properties` file in the IGV directory.  
  
I'm using IGV 2.8.13 downloaded from the IGV webpage, on Ubuntu 20, with the bundled Java environment.

Here is the script I'm using:
```
new
snapshotDirectory /home/myUserName/someDirectory/igv/
goto all
echo
goto chr1:65,827,301
echo
goto SNAP23
echo
exit
```

IGV is capable of running all the commands, particularly, `goto all`, but when it comes to `goto chr1:65,827,301` (tried without the commas, and ranges like chr1:start-end) or `goto SNAP23`, it halts, unless, as I have mentioned, I've previously removed the `prefs.properties` from the folder, in that case it works. 

Then it creates another preference file, and it stops working again. I've tried some more commands, like `load someSession.xml`, and it works, as long as it run before it freezes, of course.
Here is the IGV command (the --module-path looks like that because Ive been using the folder IGV is distributed in to test this):

`
java -showversion --module-path="./lib" -Xmx6g @igv.args -Dapple.laf.useScreenMenuBar=true -Djava.net.preferIPv4Stack=true --module=org.igv/org.broad.igv.ui.Main --batch=/pathTo/myBatchFile.txt
`

I've also tried other amounts of RAM usage, too, with the same result.

Thanks for your attention

**Comment by helgathorv** [comment]
We're about to release 2.9 which has several improvements related to running IGV from batch scripts. Before we release it, you can try the new version from https://software.broadinstitute.org/software/igv/download_snapshot

**Comment by jrobinso** [comment]
@javiercguard This issue should be fixed in 2.9.0,  just released.
